core/store/src/main/java/uk/gov/gchq/gaffer/store/schema/Schema.java

```diff
@@ -201,7 +202,9 @@ public String getTimestampProperty() {
     @Override
     public String toString() {
         try {
-            return "Schema" + new String(toJson(true), CommonConstants.UTF_8);
+            return new ToStringBuilder(this)
+                    .append(new String(toJson(true), CommonConstants.UTF_8))
```

Comment:
No, Why would we need "schema" append within an object call schema?

**current**
`uk.gov.gchq.gaffer.store.schema.Schema@6ce139a4[{
  "edges" : {
  ...
}]`

vs

**suggestion**
`uk.gov.gchq.gaffer.store.schema.Schema@6ce139a4[schema={
  "edges" : {
  ...
}]`